assets/styles/generator.scss

```diff
@@ -200,3 +200,7 @@ h2 {
 .request-transport-medium-chooser {
     margin-top: 17px;
 }
+
+#current-company-comments-colon {
```

Comment:
This doesn't fix the issue as this not an actual ID that is in the output HTML. I would suggest looking at the page in your browser's dev tools to get a better sense of the HTML that is generated.